react/OrderOptions.tsx

```diff
@@ -1,8 +1,25 @@
 import React, { FunctionComponent } from 'react'
-import { InjectedIntlProps, injectIntl } from 'react-intl'
+import { InjectedIntlProps, injectIntl, defineMessages } from 'react-intl'
 
 import ButtonLink from './ButtonLink'
 
+const messages = defineMessages({
+  printreceiptButton: {
+    id: 'store/order.header.takeaway.printreceipt.button',
+    defaultMessage: '',
+  },
+  updateButton: { id: 'store/order.header.update.button', defaultMessage: '' },
+  myordersButton: {
```

Comment:
⛏[Nitpick] why not `printReceiptButton` and `myOrdersButton`?
I know you probably chose to maintain the same name as the ones in the message Ids, which I think is the best idea, buuut, I would change those as well. Also, that's my bad 😂